src/post.ts

```diff
@@ -205,6 +209,6 @@ export default function postCombatActions(skipDiet = false): void {
   stillsuit();
   funguySpores();
   if (globalOptions.ascending || AutumnAton.turnsForQuest() < myAdventures() + 10) {
-    AutumnAton.sendTo(autumnAtonZones);
+    AutumnAton.sendTo(maxBy(autumnatonLocations, (location) => averageAutumnatonValue(location)));
```

Comment:
If we do this, we cannot reduce down our list of locations to prioritize the upgrade locations, or do I misunderstand the libram function?

Reply:
```suggestion
    AutumnAton.sendTo((locations) => maxBy(prioritizeUpgradeLocations(locations), averageAutumnatonValue));
```

Although it feels like the prioritization logic should just be part of the `value` function

Reply:
As it stands I'm a little unsure of how to do that easily because of how I use the `averageAutumnatonValue` function in `prioritizeUpgradeLocations` itself. I'll make this smaller proposed change for now